third_party/Makefile

```diff
@@ -44,8 +44,13 @@ endif
 
 # Build perftest.
 # The version we use is the tag v4.5-0.2.
-perftest: 
+perftest: sb_micro_path
 ifneq (,$(wildcard perftest/autogen.sh))
 	cd perftest && ./autogen.sh && ./configure CUDA_H_PATH=/usr/local/cuda/include/cuda.h --prefix=$(SB_MICRO_PATH) && make -j && make install
 endif
 
+# Build FIO from commit 0313e9 (fio-3.27 tag).
+fio: sb_micro_path
```

Comment:
no need to add 'sb_micro_path' if using 'make install'

Reply:
Fixed